src/main/java/com/autotune/common/target/kubernetes/service/impl/KubernetesServicesImpl.java

```diff
@@ -403,7 +403,7 @@ public void addWatcher(CustomResourceDefinitionContext crd, Watcher watcher) {
         try {
             kubernetesClient.customResource(crd).watch(watcher);
         } catch (Exception e) {
-            new TargetHandlerException(e, "addWatcher failed!");
+            LOGGER.warn("Failed to add watcher and presuming kruize running for Remote monitoring use case. ");
```

Comment:
This is a misleading message. No Watcher means no CRD, this means REST API access only is enabled. You can still run experiments of all types but only through REST API. The message should be
```
LOGGER.warn("Failed to add Watcher. Only REST API access enabled. ");
```